docs/guides/01-getting-started/prerequisites.md

```diff
@@ -0,0 +1,212 @@
+---
+sidebar_position: 1
+pagination_next: guides/getting-started/beginning-tutorial
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Prerequisites
+
+The first step is to install [Rust] and system dependencies. Keep in mind that this setup is only needed for _developing Tauri apps_. Your end-users are not required to do any of this.
+
+## Setting Up Windows
+
+:::info
+For those using the Windows Subsystem for Linux (WSL), please refer to our [Linux specific instructions](#setting-up-linux) instead.
+:::
+
+On Windows, go to [https://www.rust-lang.org/tools/install][install rust] to install `rustup` (the Rust installer). You also need to install Microsoft Visual Studio C++ build tools. The easiest way is to install [Build Tools for Visual Studio 2022]. When asked which workloads to install, ensure "C++ build tools" and the Windows 10 SDK are selected.
```

Comment:
Hmm i think we may have to swap that around. I think rustup can default to the wrong toolchain if it can't find the build tools, gonna check that real quick.

On that note, it makes sense to swap it around for every OS. It won't look/read as nice but it makes more sense imo.

Reply:
Okay it still defaults to the correct toolchain but will show an error screen beforehand (you can dismiss it with "y" and can install it like normal)

Reply:
Turns out it didn't default to the msvc toolchain for others on discord who didn't have ms build tools installed so we really need to swap them around.

Reply:
Alright 👍🏻

Reply:
Updated this now and improved the headings, give a look and let me know what you think